test/system/admin/admin_jobs_test.rb

```diff
@@ -21,8 +21,9 @@ def test_administrator_can_schedule_jobs_on_demand
     assert_text('InvoiceCreationJob')
     assert_text('InvoiceCancellationJob')
     assert_text('DailySummaryJob')
+    assert_text('DailyBroadcastAuctionsJob')
 
-    assert(page.has_button?('Run', count: 4))
+    assert(page.has_button?('Run', count: 5))
```

Comment:
Seems like magic number. What is it responsible for? Can we avoid it?

Reply:
It has some unrelated reasons to fail, therefore it's quite brittle: adding a new job immediately makes it failing. 
Based on its name `administrator_can_schedule_jobs_on_demand`, it should fail if an admin cannot schedule jobs anymore, which is a false statement if I just add a new job.

So something is wrong here. Perhaps as a first step, `ApplicationJob.descendants.count` might be a more readable alternative for that magic value...

It's indeed seems like a topic of a new PR.